perf_test/sparse/KokkosSparse_sptrsv_superlu.cpp

```diff
@@ -796,8 +796,8 @@ int main(int argc, char **argv) {
   }
 
   {
-    using scalar_t = double;
-    //using scalar_t = Kokkos::complex<double>;
+    //using scalar_t = double;
```

Comment:
Can we pick this based on what scalars are defined ?

Reply:
I will modify this perf_test to exercise both the double and complex types.

Reply:
I think it should also be what is enabled in ETI at compile time.

Reply:
Got it! Thanks!